blpapi.ts

```diff
@@ -0,0 +1,728 @@
+/// <reference path="./typings/tsd.d.ts" />
+
+'use strict';
+
+import assert = require('assert');
+import events = require('events');
+import util = require('util');
+
+import Promise = require('bluebird');
+import debug = require('debug');
+import _ = require('lodash');
+
+// Somewhat hacky but needed to import the 'blpapijs' module.
+import path = require('path');
```

Comment:
nit: since `path` is node core module, it should bundle with `assert, events & util`